### PR TITLE
review: fix: Ignore Qodana action exit code

### PR DESCRIPTION
**Pull request by I-Al-Istannen**
This PR just ignores failures in the qodana action (as its exit codes were a bit weird) and instead relies on the jq buildbreaker.

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -167,6 +167,7 @@ jobs:
           results-dir:  "${{ github.workspace }}/result/master"
       - name: Qodana - Code Inspection
         uses: JetBrains/qodana-action@cd7dc31b49427ce740ebf6255319b267801d08cf # renovate: tag=v3.2.1
+        continue-on-error: true
```
Comment (algomaster99) [inline .github/workflows/tests.yml:170]: Doesn't this make usage of qodana-action redundant? Even if it fails, we shall ignore the exit code and still move on with merging a PR.

Comment (I-Al-Istannen) [reply .github/workflows/tests.yml:170]: Not quite, as the print (short) step later does the actual checking and breaks the build.

**Review by slarse — APPROVED** [review]
LGTM, will merge.

**Comment by algomaster99** [comment]
@monperrus This PR has changes that need to be excluded from https://github.com/INRIA/spoon/pull/4298. Please merge it.

**Comment by MartinWitt** [comment]
@I-Al-Istannen thanks for the fix.

**Comment by slarse** [comment]
I of course meant LGTM, will forget to merge.

**Comment by monperrus** [comment]
Thanks for the great group work.

**Pull request merged**